### ISSUE TITLE
Permits to use doorcat on Entreprise Github instance

### ISSUE THREAD

**Issue by edimitchel** [antfu/github-doorcat#17]
Hi @antfu !

I'm working with a self-hosted Enterprise Github instance, is it possible to configurate the endpoint?
Instead of github.com, use the custom domain. And this could be a parameter of the webex.

What I see for the moment that should be being customized:
https://github.com/antfu/github-doorcat/blob/main/src/background/contentScriptHMR.ts#L13
https://github.com/antfu/github-doorcat/blob/main/src/contentScripts/fetch.ts#L15
https://github.com/antfu/github-doorcat/blob/main/src/contentScripts/fetch.ts#L20

**Comment by edimitchel** [comment]
I'll give a try by cloning the repo and try it in dev mode

**Comment by antfu** [comment]
Could do, but I am mostly concerned about that if Chrome Store allows that 🤔

**Comment by edimitchel** [comment]
And using the [Options page](https://user-images.githubusercontent.com/11247099/126741653-43125b62-6578-4452-83a7-bee19be2eaa2.png) ?

**Comment by antfu** [comment]
The problem here is the extension permission, which need to be included in the manifest statically, while have **/* could make the reviewing much harder.

**Comment by edimitchel** [comment]
Yes, understood.. :/ And what if it includes github inside the url ?

**Comment by antfu** [comment]
I see there might be a solution here:

- https://github.com/sindresorhus/refined-github/blob/32abfaaa4696053fa956f12b0186339e80f19292/distribution/manifest.json#L25-L27
- https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/optional_permissions
- https://github.com/sindresorhus/refined-github/issues/3543

![image](https://user-images.githubusercontent.com/11247099/133213722-9b205103-d640-44e3-b68d-372f334eb0bd.png)

While I don't have access to a GitHub Enterprise instance, I will need to leave this to you to test out and sending a PR.

**Comment by edimitchel** [comment]
The package `webext-domain-permission-toggle` permits it, I will try that
